cmd/kubeadm/app/util/system/kernel_validator.go

```diff
@@ -78,7 +78,6 @@ func (k *KernelValidator) Validate(spec SysSpec) (error, error) {
 
 // validateKernelVersion validates the kernel version.
 func (k *KernelValidator) validateKernelVersion(kSpec KernelSpec) error {
-	glog.V(1).Info("Validating kernel version")
```

Comment:
You could just bump they verbosity to debug levels

Reply:
i think this is just duplicate output,.